### PR TITLE
python312Packages.vobject: 0.9.7 -> 0.9.8

### PR DESCRIPTION
**Pull request by natsukium**
Diff: https://github.com/py-vobject/vobject/compare/v0.9.7...v0.9.8

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [x] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by natsukium** [comment]
Result of `nixpkgs-review pr 353145` at 21d6e5c7 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>:fast_forward: 1 package marked as broken and skipped:</summary>
  <ul>
    <li>syncall</li>
  </ul>
</details>
<details>
  <summary>:x: 6 packages failed to build:</summary>
  <ul>
    <li>odoo15</li>
    <li>odoo16</li>
    <li>odoo17</li>
    <li><a href="https://gist.github.com/natsukium/55e2c7b8b3995af4c69fcaa3cca249d5">radicale2</a></li>
    <li><a href="https://gist.github.com/natsukium/e0fc3a6137a447a06b3fbb92b7cbeb6d">weblate</a></li>
    <li><a href="https://gist.github.com/natsukium/6c51ba48a6f59a1599796143a2de2b0c">weblate.static</a></li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 40 packages built:</summary>
  <ul>
    <li>aerc</li>
    <li>calendar-cli</li>
    <li>carddav-util</li>
    <li>errands</li>
    <li>etesync-dav</li>
    <li>gcalcli</li>
    <li>gtg</li>
    <li>home-assistant-component-tests.caldav</li>
    <li>khard</li>
    <li>lbdb</li>
    <li>mailman-web</li>
    <li>odoo</li>
    <li>paperless-ngx</li>
    <li>pretalx</li>
    <li>pretalx.static</li>
    <li>pretix</li>
    <li>python311Packages.caldav</li>
    <li>python311Packages.django-ckeditor</li>
    <li>python311Packages.django-extensions</li>
    <li>python311Packages.django-hierarkey</li>
    <li>python311Packages.etesync</li>
    <li>python311Packages.radicale-infcloud</li>
    <li>python311Packages.translate-toolkit</li>
    <li>python311Packages.translation-finder</li>
    <li>python311Packages.vobject</li>
    <li>python311Packages.weblate-language-data</li>
    <li>python312Packages.caldav</li>
    <li>python312Packages.django-ckeditor</li>
    <li>python312Packages.django-extensions</li>
    <li>python312Packages.django-hierarkey</li>
    <li>python312Packages.etesync</li>
    <li>python312Packages.radicale-infcloud</li>
    <li>python312Packages.translate-toolkit</li>
    <li>python312Packages.translation-finder</li>
    <li>python312Packages.vobject</li>
    <li>python312Packages.weblate-language-data</li>
    <li>radicale</li>
    <li>vcs_query</li>
    <li>visidata</li>
    <li>xandikos</li>
  </ul>
</details>

File: pkgs/development/python-modules/vobject/default.nix [hunk 1]
```diff
@@ -3,29 +3,42 @@
...
+  ];
+
+  pythonImportsCheck = [ "vobject" ];
+
+  nativeCheckInputs = [ pytestCheckHook ];
+
+  pytestFlagsArray = [ "tests.py" ];
 
   meta = with lib; {
     description = "Module for reading vCard and vCalendar files";
```
Comment (dotlambda) [inline pkgs/development/python-modules/vobject/default.nix:40]:
```suggestion
  meta = with lib; {
    changelog = "https://github.com/py-vobject/vobject/releases/tag/v${version}";
    description = "Module for reading vCard and vCalendar files";
```

**Comment by dotlambda** [comment]
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 353145`

---
### `x86_64-linux`
<details>
  <summary>:fast_forward: 2 packages marked as broken and skipped:</summary>
  <ul>
    <li>syncall</li>
    <li>syncall.dist</li>
  </ul>
</details>
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>radicale2</li>
    <li>radicale2.dist</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 80 packages built:</summary>
  <ul>
    <li>aerc</li>
    <li>calendar-cli</li>
    <li>calendar-cli.dist</li>
    <li>carddav-util</li>
    <li>errands</li>
    <li>etesync-dav</li>
    <li>etesync-dav.dist</li>
    <li>gcalcli</li>
    <li>gcalcli.dist</li>
    <li>gtg</li>
    <li>home-assistant-component-tests.caldav</li>
    <li>khard</li>
    <li>khard.dist</li>
    <li>lbdb</li>
    <li>mailman-web</li>
    <li>mailman-web.dist</li>
    <li>odoo</li>
    <li>odoo.dist</li>
    <li>odoo15</li>
    <li>odoo15.dist</li>
    <li>odoo16</li>
    <li>odoo16.dist</li>
    <li>odoo17</li>
    <li>odoo17.dist</li>
    <li>paperless-ngx</li>
    <li>pretalx</li>
    <li>pretalx.dist</li>
    <li>pretalx.static</li>
    <li>pretix</li>
    <li>pretix.dist</li>
    <li>python311Packages.caldav</li>
    <li>python311Packages.caldav.dist</li>
    <li>python311Packages.django-ckeditor</li>
    <li>python311Packages.django-ckeditor.dist</li>
    <li>python311Packages.django-extensions</li>
    <li>python311Packages.django-extensions.dist</li>
    <li>python311Packages.django-hierarkey</li>
    <li>python311Packages.django-hierarkey.dist</li>
    <li>python311Packages.etesync</li>
    <li>python311Packages.etesync.dist</li>
    <li>python311Packages.radicale-infcloud</li>
    <li>python311Packages.radicale-infcloud.dist</li>
    <li>python311Packages.translate-toolkit</li>
    <li>python311Packages.translate-toolkit.dist</li>
    <li>python311Packages.translation-finder</li>
    <li>python311Packages.translation-finder.dist</li>
    <li>python311Packages.vobject</li>
    <li>python311Packages.vobject.dist</li>
    <li>python311Packages.weblate-language-data</li>
    <li>python311Packages.weblate-language-data.dist</li>
    <li>python312Packages.caldav</li>
    <li>python312Packages.caldav.dist</li>
    <li>python312Packages.django-ckeditor</li>
    <li>python312Packages.django-ckeditor.dist</li>
    <li>python312Packages.django-extensions</li>
    <li>python312Packages.django-extensions.dist</li>
    <li>python312Packages.django-hierarkey</li>
    <li>python312Packages.django-hierarkey.dist</li>
    <li>python312Packages.etesync</li>
    <li>python312Packages.etesync.dist</li>
    <li>python312Packages.radicale-infcloud</li>
    <li>python312Packages.radicale-infcloud.dist</li>
    <li>python312Packages.translate-toolkit</li>
    <li>python312Packages.translate-toolkit.dist</li>
    <li>python312Packages.translation-finder</li>
    <li>python312Packages.translation-finder.dist</li>
    <li>python312Packages.vobject</li>
    <li>python312Packages.vobject.dist</li>
    <li>python312Packages.weblate-language-data</li>
    <li>python312Packages.weblate-language-data.dist</li>
    <li>radicale (radicale3)</li>
    <li>radicale.dist (radicale3.dist)</li>
    <li>vcs_query</li>
    <li>visidata</li>
    <li>visidata.dist</li>
    <li>weblate</li>
    <li>weblate.dist</li>
    <li>weblate.static</li>
    <li>xandikos</li>
    <li>xandikos.dist</li>
  </ul>
</details>

**Comment by dotlambda** [comment]
> 2 packages failed to build

radicale2 will be dropped in #360284

**Pull request merged**